src/host.js

```diff
@@ -50,13 +60,16 @@ export default class Host extends Port {
 			super.open();
 		});
 	}
-	static createIFrame(src) {
+	static createIFrame(src, frameId) {
 		var iframe = document.createElement('iframe');
 		iframe.width = '100%';
```

Comment:
Let's set height here as well (if they set the option) instead of in `changeHeight`.